src/models/post.ts

```diff
@@ -0,0 +1,62 @@
+export class Post {
+
+    location: any; 
+    timestamp: any;
+    expiration: any;
+    description: string;
+    image: string;
```

Comment:
image should be a list of string since we may want user to upload multiple photos

Reply:
makes sense